server/model/monitor.js

```diff
@@ -34,6 +34,163 @@ const rootCertificates = rootCertificatesFingerprints();
  *      2 = PENDING
  *      3 = MAINTENANCE
  */
+
```

Comment:
New monitors need to be in their own file. Please see the DNS monitor for further context